### PR TITLE
Adds documentation about defining rollover action in ILM

### PR DESCRIPTION
**Pull request by colings86**

### CODE REVIEW

File: x-pack/docs/en/ilm/using-policies-rollover.asciidoc [hunk 1]
```diff
@@ -1,7 +1,121 @@
...
+<1> We name the first index `test-000001` and the rollover action will 
+increment the sufix number for each newly created index
+<2> Set this initial index to be the write index for this alias.
+
+This first index defines an alias `test-alias` which can then be used for
+indexing data. The initial `test-000001` index is set as the "write index" for
+this alias and as per the `index.lifecycle.rollover_alias` setting defined in
+the index template above the "write index" for this alias will be moved to the
+newly created index on each rollover to allow indexing to not be interrupted as
+indexes become "full" and are rolled over.
```
Comment (jakelandis) [inline x-pack/docs/en/ilm/using-policies-rollover.asciidoc]: grammar nitpick:

This first index defines an alias `test-alias` which can then be used for
indexing data. The initial `test-000001` index is set as the "write index" for
this alias and as per the `index.lifecycle.rollover_alias` setting defined in
the index template above. The "write index" for this alias will be moved to the
newly created index (e.g. `test-000002`, `test-000003`, etc.) on each rollover to 
allow indexing to not be interrupted as indexes become "full".

**Comment by jakelandis** [comment]
non authoritative LGTM (one minor grammar nitpick)

**Review by debadair — COMMENTED** [review]
I left some comments--mostly editorial/organizational. I have some more general editorial recommendations--not sure if they belong here, or elsewhere?

**Comment by colings86** [comment]
@debadair Thanks for the review. I incorporated all your comments. I also reworded the last paragraph so hopefully its not so redundant as before but let me know if you think it can be improved further.

**Comment by colings86** [comment]
@debadair In terms of the other more general editorial notes, if they apply to this change itself then feel free to add them here and I'll incorporate those changes too. If they are not directly related to the rollover documentation then maybe we can chat offline to discuss them so I can apply the comments to future PRs?

**Review by debadair — APPROVED** [review]
Left a couple minor nits, but LGTM.

**Comment by colings86** [comment]
Thanks for reviewing @debadair

**Pull request merged**